Form/Extension/DateTypeExtension.php

```diff
@@ -59,7 +59,7 @@ public function finishView(FormView $view, FormInterface $form, array $options)
     /**
      * {@inheritdoc}
      *
-     * @todo Remove it when bumping requirements to SF 2.7+
+     * @deprecated Remove it when bumping requirements to SF 2.7+
```

Comment:
just for insight: have a look into https://insight.sensiolabs.com/projects/0a6dbd4c-714b-47a4-b662-254cdf6ec208/analyses/2 ...
TODO comments are left in the code when a feature (or a bug) isn't completely developed (or fixed). You should complete the implementation and remove the comment.